tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_mobilenet_gptq.ipynb

```diff
@@ -0,0 +1,485 @@
+  },
+  {
+   "cell_type": "markdown",
+   "id": "028112db-3143-4fcb-96ae-e639e6476c31",
+   "metadata": {
+    "id": "028112db-3143-4fcb-96ae-e639e6476c31"
+   },
+   "source": [
+    "Create a generator for the representative dataset.\n",
+    "The representative dataset is used for collecting statistics on the inference outputs of all layers in the model.\n",
+    "The batch size and the number of iterations determine the size of the representative dataset (n_iter x batch_size).\n",
+    "In this example we set `batch_size = 50` and `n_iter = 10`, resulting in a total of 500 representative images.\n",
+    "\n",
+    "GPTQ is a gradient-based optimization process, which requires representative dataset to perform inference and compute gradients. It is possible to define a separate representative dataset in addition to the one used for the PTQ statistics collection. A possible reason to do so is, for example, to use a larger dataset in the optimization process.\n",
+    "\n",
+    "In this tutorial we use the same representative dataset for both statistics collection and GPTQ. \n",
+    "A complete pass through the representative dataset generator constitutes an epoch (batch_size x n_iter samples).\n",
+    "In this example we use the same dataloader iterator for all epochs, i.e. different images are used in different epochs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ed56f505-97ff-4acb-8ad8-ef09c53e9d57",
+   "metadata": {
+    "id": "ed56f505-97ff-4acb-8ad8-ef09c53e9d57"
+   },
+   "outputs": [],
+   "source": [
+    "batch_size = 50\n",
+    "n_iter = 10\n",
+    "\n",
+    "dataloader = DataLoader(dataset, batch_size=batch_size, shuffle=True)\n",
+    "dataloader_iter = iter(dataloader)\n",
+    "\n",
+    "def representative_dataset_gen():\n",
+    "    for _ in range(n_iter):\n",
+    "        yield [next(dataloader_iter)[0]]\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a8b486a-ca39-45d9-8699-f7116b0414c9",
+   "metadata": {
+    "id": "8a8b486a-ca39-45d9-8699-f7116b0414c9"
+   },
+   "source": [
+    "## Model Gradient-Based Post-Training quantization using MCT\n",
+    "\n",
+    "This is the main part in which we quantize and our model.\n",
+    "\n",
+    "Next, we create a **GPTQ configuration** with possible GPTQ optimization options (such as the number of epochs for the optimization process). \n",
+    "MCT will quantize the model and start the GPTQ process to optimize the model's parameters and quantization parameters.\n",
+    "\n",
+    "In addition, we need to define a `TargetPlatformCapability` object, representing the HW specifications on which we wish to eventually deploy our quantized model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2edacb5b7779e4d8",
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "from model_compression_toolkit.gptq.common.gptq_constants import REG_DEFAULT\n",
```

Comment:
No need to import and pass this constant, it is the default value in the config anyway (I know it is like that in the Keras tutorial, it is not needed there either)